Source/Controllers/Preferences/Panes/SPFilePreferencePane.m

```diff
@@ -33,23 +33,36 @@
 
 @import Firebase;
 
+typedef struct {
+    NSString *title;
+    BOOL canChooseFiles;
+    BOOL canChooseDirectories;
+    BOOL allowsMultipleSelection;
+    BOOL isForStaleBookmark;
+    NSString *fileName;
+} PanelOptions;
+
 @interface SPFilePreferencePane ()
 - (void)_refreshBookmarks;
 
+@property (readwrite, strong) NSMutableArray<NSDictionary<NSString *, id> *> *bookmarks;
+@property (readwrite, strong) NSMutableArray<NSString *> *staleBookmarks;
+
 @end
 
 @implementation SPFilePreferencePane
 
-@synthesize bookmarks;
+@synthesize bookmarks, staleBookmarks, staleLabel;
 
 - (instancetype)init
 {
 	self = [super init];
 	
 	if (self) {
 		fileNames = [[NSMutableArray alloc] init];
-		bookmarks = [[NSMutableArray alloc] init];
-        
+        bookmarks = [[NSMutableArray alloc] init];
+        staleBookmarks = [[NSMutableArray alloc] init];
+
```

Comment:
Spacing looks funky here
Perhaps we may need to lint all the files in the repo? Perhaps at a point when we have no open PRs we could do this, @Kaspik?